src/firebolt/async_db/cursor.py

```diff
@@ -366,6 +377,35 @@ async def _validate_set_parameter(self, parameter: SetParameter) -> None:
         # set parameter passed validation
         self._set_parameters[parameter.name] = parameter.value
 
+    def _validate_ss_async_settings(
+        self,
+        parameters: Sequence[Sequence[ParameterType]],
+        queries: List[Union[SetParameter, str]],
+        skip_parsing: bool = False,
+        async_execution: Optional[bool] = False,
+    ) -> None:
+        if async_execution and self._set_parameters.get("use_standard_sql", "1") == "0":
+            raise AsyncExecutionUnavailableError(
+                "It is not possible to execute queries asynchronously if "
+                "use_standard_sql=0."
+            )
+        if parameters and skip_parsing:
+            logger.warning(
+                "Query formatting parameters are provided but skip_parsing "
+                "is specified. They will be ignored."
+            )
+
+        # Allow users to manually skip parsing for performance improvement.
```

Comment:
I think this comment got lost?

Reply:
Actually, this entire function is an orphan. I killed it.

Reply:
Oops!